blank-canvas-blocks/sass/elements/_links.scss

```diff
@@ -2,29 +2,23 @@
 
 a:not(.wp-block-button__link) {
 	cursor: pointer;
-	color: var( --wp--style--color--link, var(--wp--custom--color--foreground) );
+	text-underline-offset: 2px;
 	text-decoration: none;
-	border-bottom: 1px solid transparent;
 
 	&:hover,
 	&:focus {
-		border-bottom-color: currentColor;
+		text-decoration: underline;
 	}
 }
 
 // Links that are in the content
 .entry-content a:not(.wp-block-button__link) {
-	border-bottom: 1px solid currentColor;
-	text-decoration: none;
+	text-decoration: underline;
```

Comment:
Since it's top of mind (#3476), this will interfere with the file block: 

<img width="424" alt="Screen Shot 2021-03-16 at 4 17 51 PM" src="https://user-images.githubusercontent.com/5375500/111374184-0987b780-865a-11eb-9eb9-fe8bf03a8dcf.png">

Reply:
Hrm.  I **already** didn't like including the :not(.wp-block-button__link), I'm not keen to add more and actually would maybe prefer to refactor that existing :not() out and instead target .wp-block-button__link directly in the button block styles, and the file block's button class to set the text-decoration to none.

Thoughts on the best way to address?

Reply:
I like your idea! If I understood it correctly, I tried it in https://github.com/Automattic/themes/pull/3478/commits/3c05ff6c48a2f4d4ee53ffc84abfeb2814a58ce3. It plays nicely with #3476 too since we're making use of the `button-main-styles` mixin.

Reply:
I don't think this is going to work for the editor either, since there's no `.entry-content` element in the editor. I think you can use `.site-blocks` instead.

Reply:
This is what I see in the editor: <img width="415" alt="Screenshot 2021-03-17 at 17 41 30" src="https://user-images.githubusercontent.com/275961/111512774-03700480-8748-11eb-8bcd-a2165c3ff32b.png">

Reply:
Turns out `wp-site-blocks` is no good as it wraps the whole site. In the end I had to use two classes - one for the post editor and one for the frontend and site editor.